src/AaveV3StrategicAssets_20230606/README.md

```diff
@@ -0,0 +1,207 @@
+# Streategic Assets Manager Information
+
+## Main Idea
+
+The main idea of this new contract is for the DAO to be able to handle strategic assets (assets that will earn rewards, or assets that can be used to
+vote on different protocols, for example) without the need to go through the full governance flow every week.
+
+Take the following example: the DAO wants to hold veBAL as part of its strategy for the long term. To get the most out of veBAL, the DAO should "re-lock"
+its holdings on a weekly basis. This is a very tedious approach and can lead to a lot of voter fatigue. Because of this, the StrategicAssetsManager contract
+can be governed by the DAO, while some functions can be invoked by an allowed guardian, acting on the role of Asset Manager.
+
+## Functionality
+
+#### StrategicAssetsManager.sol
+
+```
+  function addVeToken(
+    address underlying,
+    address veToken,
+    address warden,
+    uint256 lockDuration,
+    address initialDelegate,
+    bytes32 spaceId
+  ) external onlyOwner
+```
+
+This function, only governed by the DAO, sets the information needed in order to handle a veToken and all other functionality related to it.
+Without this information, the rest of the functionality will revert.
+
+Example: underlying would be B-80BAL-20WETH, veToken would be veBAL.
+Warden is a protocol by Paladin: https://app.warden.vote/boosts/ where users can boost their rewards or offer to lock their tokens for a price.
+Lock duration is the amount of time the veToken will be locked for. In the case of veBAL, max is 1 year. It can vary across different tokens.
+The initial delegate, is somebody who can vote. SpaceID is the snapshot ID for voting, which veBAL uses. veCRV votes via an Aragon contract.
+
+`function removeVeToken(address underlying) external onlyOwner`
+
+Removes the ability to interact with a certain veToken.
+
+`function addSdToken(address underlying, address sdToken, address depositor) external onlyOwner`
+
+SD tokens are StakeDAO tokens: https://stakedao.gitbook.io/stakedaohq/
+For them, the underlying, the sdToken address of the token, and the address where to deposit the underlying to receive sdToken.
+
+`function removeSdToken(address underlying) external onlyOwner`
+
+Removes the ability to interact with a certain sdToken.
+
+`function withdrawERC20(address token, address to, uint256 amount) external onlyOwner`
+
+Sends ERC20 tokens to an address. Withdrawal mechanism.
+
+`function setStrategicAssetsManager(address _manager) external onlyOwner`
+
+Adds manager role.
+
+`function removeStrategicAssetManager() external onlyOwner`
```

Comment:
Maybe I'd change the naming here since the contract itself is the `StrategicAssetsManager` there could be a bit of confusion (despite the s)

Reply:
Docs were outdated, thanks for pointing out.